score-lang-compiler/src/main/java/org/openscore/lang/compiler/modeller/ExecutableBuilder.java

```diff
@@ -111,25 +111,34 @@ public Executable transformToExecutable(ParsedSlang parsedSlang, String execName
                 if(!executableRawData.containsKey(SlangTextualKeys.WORKFLOW_KEY)){
                     throw new RuntimeException("Error compiling " + parsedSlang.getName() + ". Flow: " + execName + " has no workflow property");
                 }
-                LinkedHashMap<String, Map<String, Object>> workFlowRawData;
+                List<Map<String, Map<String, Object>>> workFlowRawData;
```

Comment:
I think that the best representation is as a `list<tuple<string, map>` and since you don't have tuple in java you might use map.entry or guava's pair that extends it. It will save you all of the redundant iterator next to get the first key each time.